frontend/src/main.ts

```diff
@@ -49,5 +50,9 @@ bootstrapApplication(AppComponent, {
     },
     provideAnimations(),
     provideHttpClient(withInterceptorsFromDi()),
+    {
+      provide: BASE_PATH,
+      useValue: environment.backend_url.replace('/api/v1', ''),
+    },
```

Comment:
It would be great to have a short comment here explaining why this is done here.

Reply:
Done.